src/librustc_borrowck/borrowck/mir/dataflow/sanity_check.rs

```diff
@@ -116,20 +116,26 @@ fn each_block<'a, 'tcx, O>(tcx: TyCtxt<'a, 'tcx, 'tcx>,
                                      repr::BorrowKind::Shared,
                                      ref peeking_at_lval) = *rvalue {
                 // Okay, our search is over.
-                let peek_mpi = move_data.rev_lookup.find(peeking_at_lval);
-                let bit_state = sets.on_entry.contains(&peek_mpi);
-                debug!("rustc_peek({:?} = &{:?}) bit_state: {}",
-                       lvalue, peeking_at_lval, bit_state);
-                if !bit_state {
-                    tcx.sess.span_err(span, &format!("rustc_peek: bit not set"));
+                match move_data.rev_lookup.find(peeking_at_lval) {
```

Comment:
(Putting pedantic hat on): technically the `span_err` are for unit testing; see e.g. https://github.com/rust-lang/rust/blob/34fd68668152530dcd1d00865fa8514461b895d7/src/test/compile-fail/mir-dataflow/uninits-1.rs

(making the message more meaningful might be nice nonetheless, but we would want to balance that against the effort required to maintain the unit tests...)